src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/stateful_sdpa_fusion.cpp

```diff
@@ -53,9 +53,12 @@ StatefulSDPAFusion::StatefulSDPAFusion() {
     auto concat_k = makePattern<opset1::Concat>({gather_input_k, cur_k}, {{"axis", axis_seq_len}});
     auto concat_v = makePattern<opset1::Concat>({gather_input_v, cur_v}, {{"axis", axis_seq_len}});
 
-    auto multi_query_bcst = [](std::shared_ptr<Node> kv) {
-        auto reshape_kv = wrap_type<opset6::Reshape>({kv, any_input()});
-        auto unsqueeze_kv = makePattern<opset1::Unsqueeze>({kv, any_input()});
+    std::shared_ptr<Node> reshape_k, reshape_v, unsqueeze_k, unsqueeze_v;
+    std::shared_ptr<Node> computed_bcst_k, computed_bcst_v, multiply_k, multiply_v;
+    auto multi_query_bcst = [](const std::shared_ptr<Node>& kv, std::shared_ptr<Node>& reshape_kv,
+        std::shared_ptr<Node>& unsqueeze_kv, std::shared_ptr<Node>& computed_bcst, std::shared_ptr<Node>& multiply_kv) {
```

Comment:
NIT. Isn't it more intuitive to return a tuple with the original return type and all the newly added pointers? Something like that:
```cpp
    std::shared_ptr<Node> reshape_k, reshape_v, unsqueeze_k, unsqueeze_v;
    std::shared_ptr<Node> computed_bcst_k, computed_bcst_v, multiply_k, multiply_v;
    std::shared_ptr<Node> mq_reshape_k;
    std::tie(mq_reshape_k, reshape_k, unsqueeze_k, computed_bcst_k, multiply_k) = multi_query_bcst(concat_k);
```

Reply:
Updated.